### PR TITLE
Mostly code style fixes and removal of code that isn't needed

### PR DESCRIPTION
**Pull request by 1DC**

### CODE REVIEW

File: src/Site.php [hunk 1]
```diff
@@ -77,54 +77,54 @@ private function loadRow($row) {
...
 
             $value = substr($line, strpos($line,":")+1);
 
             switch ($parameter) {
                 case "postcode":
-                error_log("*".$parameter."*");
-                $this->postcode = $value;
-                $updated = TRUE;
-                break;
+                    error_log("*".$parameter."*");
```
Comment (afoldesi-gds) [inline src/Site.php]: Would you mind adding a few spaces around those dots :)

Comment (1DC) [inline src/Site.php]: Whoops, didn't notice those when I went through it all. Fixed. :)

File: src/SmsResponse.php [hunk 2]
```diff
@@ -116,31 +117,24 @@ public function getTemplateForOs($os) {
         if (empty($os)) {
             return $config->values['notify']['creds-unknown'];
         }
+
         switch ($os) {
             case (preg_match("/(mac|OSX|apple)/i", $os) ? true : false):
                 return $config->values['notify']['creds-mac'];
-                break;
```
Comment (afoldesi-gds) [inline src/SmsResponse.php]: I normally leave these in so it doesn't get forgotten when the logic changes, but it does look better without it doesn't it.

Comment (1DC) [inline src/SmsResponse.php]: I don't think you should be adding additional lines of code to cater to forgetful people who work on it in future. :P

Comment (1DC) [inline src/SmsResponse.php]: I don't think you should be adding additional lines of code to cater to forgetful people who work on it in future. :P

Comment (1DC) [inline src/SmsResponse.php]: Updated although I disagree with that choice. :)

**Comment by 1DC** [comment]
@afoldesi-gds updated. Can you review again, please?

**Comment by afoldesi-gds** [comment]
Will be merging this a bit later - prob tomorrow

**Comment by 1DC** [comment]
@afoldesi-gds no worries, thanks! 👍

**Pull request merged**